packages/graphql-lowcode/src/generate/generateGraphqlQueries.ts

```diff
@@ -22,79 +24,212 @@ interface IntrospectionQuery {
   [key: string]: any
 }
 
-interface Fragment {
-  name: string,
-  queryName: string,
-  entity: string,
-  fields: string[]
+interface Root {
+  fields: Field[],
+  kind: string,
+  [key: string]: any
 }
 
-interface QueryRoot {
+interface Entity {
```

Comment:
Better naming would be `EntityQuery` since there is not only `entityName` and `fields` but also `queryName`